plugins/modules/azure_ad_password_info.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Haiyuan Zhang, <haiyzhan@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+import datetime
+try:
+    from dateutil.relativedelta import relativedelta
+except ImportError:
+    pass
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: azure_ad_password_info
+
+version_added: "2.10"
+
+short_description: Get application info
+
+description:
+        - Get application info.
+
+options:
+    app_id:
+        description:
+            - The application ID.
```

Comment:
Get Application password info

Reply:
This is the description of the 'app_id', should not need to change!！